ci/run_ci.sh

```diff
@@ -0,0 +1,80 @@
+
+#!/bin/bash
+
+source ci/env.sh
+
+test_choice=$@                                  #to accept inputs from command line
+
+if [[ ${test_choice} != '' ]]; then
+  echo -e "Running chosen tests: "
+  for i in ${test_choice};do
+    echo -e "${i}"
+  done
+  test_list=${test_choice}
+else
+  echo "Running full test"
+  test_list=`cat ci/CI_test_list.yml`           #yml file contains the list of test scripts to run from different workloads
+  echo -e  "$test_list\n\n"                         
+fi
+
+diff_list=`git diff --name-only origin/master`
+echo -e "List of files changed : ${diff_list} \n"
+
+cat > results.markdown << EOF
+Results for e2e-benchmarking CI Tests
+Workload                | Test                           | Result | Runtime  |
+------------------------|--------------------------------|--------|----------|
+EOF
+
+
+test_rc=0
+bit=0
+
+for test in ${test_list}; do            
+
+  start_time=`date`
+
+  command=${test##*:}                                     #to extract the shell script name to run
+  directory=${test%:*}                                    #to extract the workload directory name 
+  
+  echo -e "\n======================================================================"
+  echo -e "     CI test for ${test}                    "
+  echo -e "======================================================================\n"
+
+  if [[ ( $directory == scale-perf ) && ( $bit == 0 ) ]]; #check to increment number of worker nodes by 1 and to bring it 
```

Comment:
when user will try to run only scale-test using "/rerun scale-perf/........" the scale-down will not be triggered.

Instead, check the number of worker node at the end of all / single test and if it is not equal to $ORIGINAL_WORKER_COUNT then run the scale job with SCALE=$ORIGINAL_WORKER_COUNT